SECURITY.md

```
@@ -0,0 +1,53 @@
+}
+return operations_;
+}
+// operationsBatch(start, end) at lines 170-182 has the identical bug:
+function operationsBatch(uint256 start, uint256 end) public view returns
+(OperationBatch[] memory operationsBatch_) {
+if (start > end || start >= _operationsBatchIdSet.length()) {
+revert InvalidIndexRange(start, end);
+}
+operationsBatch_ = new OperationBatch[](end - start);
+for (uint256 i = start; i < end; i++) {
+operationsBatch_[i] = _operationsBatchMap[_operationsBatchIdSet.at(i)]; //
```

markdownlint-cli2 (0.23.0)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)